ansible/cloud_providers/osp_infrastructure_deployment.yml

```diff
@@ -40,10 +33,13 @@
     - step001.2
     - create_inventory
     - create_ssh_config
+  vars:
+    osp_auth_username_member: "{{ guid }}-user"
+    osp_auth_password_member: "{{ heat_user_password }}"
   environment:
     OS_AUTH_URL: "{{ osp_auth_url }}"
-    OS_USERNAME: "{{ osp_auth_username }}"
-    OS_PASSWORD: "{{ osp_auth_password }}"
+    OS_USERNAME: "{{ osp_auth_username_member }}"
+    OS_PASSWORD: "{{ osp_auth_password_member }}"
     OS_PROJECT_NAME: "{{ osp_project_name }}"
     OS_PROJECT_DOMAIN_ID: "{{ osp_auth_project_domain }}"
     OS_USER_DOMAIN_NAME: "{{ osp_auth_user_domain }}"
```

Comment:
Should we make it cleaner, like the previous block, and have those inside each role instead of globally here ?

Reply:
We only need these for the one task in this play in order to get the instance info with `os_instance_facts`. We could move it down to the task level, but there is no role to move it into. I'm impartial on this.